libs/gamify/badges/src/lib/dto/create-badge.input.ts

```diff
@@ -0,0 +1,30 @@
+import { BadgeTier } from "@gamify/core";
+import { IsBoolean, IsEnum, IsInt, IsNotEmpty, IsOptional, IsPositive, IsUrl } from "class-validator";
+import { Type } from "class-transformer";
+
+export class CreateBadgeInput {
+
+    @IsNotEmpty()
+    name: string;
+
+    @IsInt()
+    @Type(() => Number)
+    applicationId: number;
+
+    @IsOptional()
+    @IsUrl() // TODO should be uploadable image
```

Comment:
Unsure about storing a url in a property with the name 'path'. 

Just a remark. It's not blocking this PR.

Reply:
We've already discussed this topic offline